.gitpod.yml

```diff
@@ -45,13 +45,13 @@ tasks:
       cp sample.env .env &&
       npm ci &&
       gp sync-done npm-ci &&
-      gp await-port 27017 &&
+      gp ports await 27017 &&
```

Comment:
I don't use GitPod - what does this change achieve?

Reply:
`await-port` is deprecated.